services/resources-getter.js

```diff
@@ -135,11 +135,17 @@ function ResourcesGetter(model, opts, params) {
       var orQuery = { $or: [] };
 
       _.each(model.schema.paths, function (value, key) {
+        var q = {};
+
         if (value.instance === 'String') {
-          var q = {};
           q[key] = new RegExp('.*' + params.search + '.*', 'i');
-
           orQuery.$or.push(q);
+        } else if (value.instance === 'Array') {
+          var field = _.findWhere(schema.fields, { field: key });
+          if (field && _.isArray(field.type) && field.type[0] === 'String') {
+            q[key] = new RegExp('.*' + params.search + '.*', 'i');
+            orQuery.$or.push(q);
```

Comment:
Might be a better code with factorisation of:
```
q[key] = new RegExp('.*' + params.search + '.*', 'i');
orQuery.$or.push(q);
```
instead of code duplication